monai/transforms/intensity/array.py

```diff
@@ -1137,3 +1130,120 @@ def __call__(self, img: np.ndarray) -> np.ndarray:
         return np.asarray(
             np.interp(img, reference_control_points_scaled, floating_control_points_scaled), dtype=img.dtype
         )
+
+
+class RandGibbsNoise(RandomizableTransform):
+    """
+    Naturalistic image augmentation via Gibbs artifacts. The transform 
+    randomly applies Gibbs noise to 3D MRI images. Gibbs artifacts
+    are one of the common type of type artfifacts appearing in MRI scans.
+    
+    For general information on Gibbs artifacts, please refer to:
+    https://pubs.rsna.org/doi/full/10.1148/rg.313105115
+    https://pubs.rsna.org/doi/full/10.1148/radiographics.22.4.g02jl14949
+
+
+    Args:
+        keys: 'image', 'label', or ['image', 'label'] depending on which data
+                you need to transform.
+        alpha: Parametrizes the intensity of the Gibbs noise filter applied. Takes
+            values in [0,1] with alpha = 0 acting as the identity mapping. 
+            If a length-2 list is given as [a,b] then the value of alpha will be 
+            sampled uniformly from the interval [a,b]. 0 <= a <= b <= 1.
+        prob: probability of applying the transform. 
+
+    """
+
+    def __init__(self, alpha: Union[float, List[float]] = 0.5, prob: float = 0.1,) -> None:
```

Comment:
Think most MONAI transforms will put the probability at the front. Guess you can also delete the trailing comma.

Reply:
ok, I'll fix it.